core/vm/contracts.go

```diff
@@ -83,17 +82,20 @@ func ProgressTip(ctx context.Context, currentTimestamp uint32) {
 	log.Info("Checking for TBC progression available", "utxoIndexHeight", uh, "txIndexHeight", th)
 
 	if uh != th {
-		log.Crit("TBC is in an unexpected state, utxoIndexHeight != txIndexHeight!",
+		log.Error("TBC is in an unexpected state, utxoIndexHeight != txIndexHeight!",
 			"utxoIndexHeight", uh, "txIndexHeight", th)
+		return
 	}
 
 	tipHeight, headers, err := TBCIndexer.BlockHeadersBest(ctx)
 	if err != nil {
-		log.Crit("Unable to retrieve tip headers from TBC", "err", err)
+		log.Error("Unable to retrieve tip headers from TBC", "err", err)
+		return
 	}
 	log.Info(fmt.Sprintf("TBC download status: tipHeight=%d", tipHeight))
```

Comment:
I think key/value pairs could be used for consistency here instead of using Sprintf - unless there's a reason not to? 🤔 
```suggestion
	log.Info("TBC download status", "tipHeight", tipHeight)
```